src/metrics-modeling/metricbot/grading.py

```
@@ -0,0 +1,236 @@
+    lines = [
+        header,
+        f"  Contests: {r.week_contests} predicted, {r.graded} graded, "
+        f"{r.ungradeable} ungradeable",
+    ]
+    if r.su:
+        su = r.su
+        fav = su["baseline_favorite"]
+        if su["accuracy"] is None:
+            lines += [
+                "",
+                f"  STRAIGHT UP   n/a — no decided games"
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)